model/spdk_installation.rb

```diff
@@ -7,6 +7,7 @@
 
 class SpdkInstallation < Sequel::Model
```

Comment:
Re, commit message: I think you mean  "**Add a** program to remove an spdk installation"